doc/nrf/ug_nrf7002_constrained.rst

```diff
@@ -0,0 +1,137 @@
++------------------------------------------+-----------------------------+--------------------------------------------------------------------------------------------------------------------------+
+
+NRF700X driver performance and memory fine tuning knobs
+*******************************************************
+
+The nRF700x driver provides the following software configurations to fine tune memory and performance based on the use case:
+
++------------------------------------------+------------------------------+-----------------------------------------------------------------------------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------+
+|Configuration Option                      | Values                       | Description                                                                       | Impact                                 | Purpose                                                                                                       |
++==========================================+==============================+===================================================================================+========================================+===============================================================================================================+
+| :kconfig:option:`CONFIG_WPA_SUPP`        | ``y`` or ``n``               | Enable or disable Wi-Fi Protected Access (WPA) supplicant                         | Memory savings                         | This specifies the inclusion of the WPA supplicant modu                                                       |
+|                                          |                              |                                                                                   |                                        | Disabling this flag restricts the nRF700x driver's functionality to STA scan only.                            |
++------------------------------------------+------------------------------+-----------------------------------------------------------------------------------+----------------------------------------+---------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`CONFIG_NRF700X_AP_MODE` | ``y`` or ``n``               | Enable or disable Access Point (AP) mode                                          | Memory savings                         | This specifies the inclusion of the AP mode module.                                                           |
+|                                          |                              |                                                                                   |                                        | Disabling this flag restricts the nRF700x driver's functionality to :term:`Station mode (STA)` only.          |
```

Comment:
Please confirm if the :term:`Station mode (STA)` is linking to the correct term in the glossary.
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/glossary.html#term-Station-mode-STA

Reply:
Yes, this is correct.